src/link/api.js

```diff
@@ -57,6 +57,11 @@ export const fulltextSearch = ( lang, term, callback ) => {
 	return request( url, handler( callback ) );
 };
 
+export const getArticleText = ( lang, title, callback ) => {
```

Comment:
Do we still need that if we're using `wikipediapreview.getSections()`?

Reply:
Right removing, thanks